test_project/settings.py

```diff
@@ -224,3 +226,5 @@
 # File to save revoked tasks across workers restart
 CELERY_WORKER_STATE_DB = os.path.join(TMP_DIR, 'celery_states')
 CELERY_BEAT_SCHEDULE_FILENAME = os.path.join(TMP_DIR, 'celerybeat_schedule')
+
+NEWSLETTERS_MAILCHIMP_APIKEY = '78ce5b4dfc49245cacd9fa255acf14d0'
```

Comment:
This is insecure, never store passwords inside the code

Reply:
sorry, already delete this part